src/Tests/Service/U2fServiceAuthenticationVerificationTest.php

```diff
@@ -252,4 +268,96 @@ class_exists('u2flib_server\U2F');
             [235789],
         ];
     }
+
+    /**
+     * @test
+     * @group authentication
+     * @dataProvider expectedDeviceErrors
+     *
+     * @param int $deviceErrorCode
+     */
+    public function it_handles_expected_device_errors($deviceErrorCode)
+    {
+        $keyHandle = 'key-handle';
+        $challenge = 'challenge';
+
+        $request = new \Surfnet\StepupU2fBundle\Dto\SignRequest();
```

Comment:
Because there are two SignRequest classes, one in the bundle, one in the lib. Could import them with aliases, but this is clear as well imo.